packages/sdk-rtl/src/paging.ts

```diff
@@ -34,7 +34,7 @@ import { IAPIMethods } from './apiMethods'
 import { BaseTransport } from './baseTransport'
 
 export const LinkHeader = 'link'
-export const TotalCountHeader = 'X-Total-Count'
```

Comment:
I feel like I see the capitalized form of http headers more often: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

also, server/client should consider them case-insensitive: https://stackoverflow.com/a/5259004/13802304

Reply:
Yeah, it's a problem with the headers handling in JS. I responded via chat that I think we need to support both the Title case and lowercase at least.